lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,12 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
+    def update_destination(destination_branch, remote = 'origin')
       origin_branch = current_branch
       run_command_with_label "git checkout #{destination_branch}"
-      run_command_with_label "git pull origin #{destination_branch}"
+      run_command_with_label "git pull #{remote} #{destination_branch}"
       run_command_with_label "git checkout #{origin_branch}"
+      run_command_with_label "git rebase #{destination_branch}"
```

Comment:
This seems like a bit of overkill since this method is only used in 1 place and the `update_destination` method is only called from `git_reflow.rb`. I have already abstracted out most of the logic for `base_branch`, `feature_branch`, and `remote`. 

Nonetheless, I will add this. Do we need tests for this method?